draft-webtransport-http2.md

```diff
@@ -5,7 +5,7 @@ docname: draft-webtransport-http2-latest
 
```

Comment:
Line 410: "If the Connect Stream..." - Isn't this different from what's in WT? Doesn't it say that everything is gone once the WT connect stream is gone?

Reply:
Yup, will put the in the issue above, good catch